gensim/test/test_keras_integration.py

```diff
@@ -59,7 +59,7 @@ def testEmbeddingLayerCosineSim(self):
         embedding_b = embedding_layer(input_b)
         similarity = dot([embedding_a, embedding_b], axes=2, normalize=True)
 
-        model = Model(input=[input_a, input_b], output=similarity)
+        model = Model([input_a, input_b], similarity)
```

Comment:
Named parameters are both safer and more descriptive. Are the new plural names (Keras 2.4.0+) backward compatible with the older 2.3.1?

If so let's use those.

Reply:
I just verified on keras 2.3.1 that both "input" and "inputs" work. Let me raise a new patch using named parameters inputs, and outputs.

For verification, I will make the changes in two-phases, one with keras restricted to 2.3.1 and other without any keras version pin (so, keras will be 2.4.3) to make sure it works for both.